appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockLineChartViewBase.java

```diff
@@ -0,0 +1,13 @@
+package com.google.appinventor.client.editor.simple.components;
+
+/**
+ * Chart View base class for Line Chart based Charts
+ */
+public abstract class MockLineChartViewBase extends MockPointChartView {
```

Comment:
The class is a superclass for the MockLineChartView and MockAreaChartView. I agree that right now it does not add much significance. This was made with 2 ideas in mind:
1. In the future, there may be some shared functionality that would apply to both Area & Line Chart views
2. This is consistent with the Android representation (which indeed uses a LineChartViewBase)

Should we remove this nonetheless?

Reply:
@ewpatton With a bit of a refactor with the constructor, this class now also makes more sense, since MockLineChartBaseDataModel can use this class as a parameter. What do you think?